scanmem.h

```diff
@@ -29,6 +30,28 @@
 #endif
 */
 
+/* from string.h in glibc */
+#ifndef strdupa
```

Comment:
Oh wait, `#ifndef` does not work for real functions,
so what if `strdupa` is a real function? `#ifndef` will be still true here.